Tuya binding compile failure in 4.0.x Branch

Hello,
I upgraded my OH install to 4.0 M2. I had the smarthomej repository installed through the UI.
The smarthomej/tuya binding disappeared, so i thought to compile it manually.

When i compile, one of the test fails with the following error:

```
[ERROR] Failures: 
[ERROR]   ConversionUtilTest.hexColorDecodeTestOld:46 expected: <240,100,50> but was: <240,100,50.196>
[INFO] 
[ERROR] Tests run: 14, Failures: 1, Errors: 0, Skipped: 0
[INFO] 
```
Any ideas on how can is this a known issue in the 4.0.x branch?

Thanks

--- a/bundles/org.smarthomej.binding.knx/src/test/java/org/smarthomej/binding/knx/internal/dpt/DPTTest.java
+++ b/bundles/org.smarthomej.binding.knx/src/test/java/org/smarthomej/binding/knx/internal/dpt/DPTTest.java
@@ -86,9 +86,9 @@ public class DPTTest {
         HSBType hsbType = (HSBType) ValueDecoder.decode("251.600", data, HSBType.class);
 
         assertNotNull(hsbType);
-        assertEquals(207, hsbType.getHue().doubleValue(), 0.1);
-        assertEquals(23, hsbType.getSaturation().doubleValue(), 0.1);
-        assertEquals(19, hsbType.getBrightness().doubleValue(), 0.1);
+        assertEquals(207, hsbType.getHue().doubleValue(), 0.0025 * 360);
+        assertEquals(23, hsbType.getSaturation().doubleValue(), 0.0025 * 100);
+        assertEquals(19, hsbType.getBrightness().doubleValue(), 0.0025 * 100);
     }
 
     @SuppressWarnings("unused")

--- a/bundles/org.smarthomej.binding.tuya/src/test/java/org/smarthomej/binding/tuya/internal/cloud/ConversionUtilTest.java
+++ b/bundles/org.smarthomej.binding.tuya/src/test/java/org/smarthomej/binding/tuya/internal/cloud/ConversionUtilTest.java
@@ -43,7 +43,7 @@ public class ConversionUtilTest extends JavaTest {
         String hex = "00008000f0ff8b";
         HSBType hsb = ConversionUtil.hexColorDecode(hex);
 
-        Assertions.assertEquals(new HSBType("240,100,50"), hsb);
+        Assertions.assertEquals(new HSBType("240,100,50.196"), hsb);
     }
 
     @Test

